packages/block-library/src/post-date/index.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-date` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-date` block on the server.
+ *
+ * @return string Returns the filtered post date for the current post wrapped inside "h5" tags.
```

Comment:
Why h5 in the PHP file, and h6 above in the React component? This will have a similar effect on screen readers as described above.

Reply:
https://github.com/WordPress/gutenberg/pull/19578#discussion_r365509200

I'll make them match after I change the element.